src/components/CommentList.jsx

```diff
@@ -0,0 +1,35 @@
+import { useEffect, useState } from "react";
+import { fetchCommentsByArticle } from "../utils/utils";
+import CommentCard from "./CommentCard";
+
+const CommentList = ({ article_id }) => {
+  const [commentList, setCommentList] = useState([]);
+  const [isLoading, setIsLoading] = useState(false);
+
+  useEffect(() => {
+    setIsLoading(true);
+    fetchCommentsByArticle(article_id).then(({ comments }) => {
+      setCommentList(comments);
+      setIsLoading(false);
+    });
+  }, []);
+
+  if (isLoading) return <h2>Comments are loading...</h2>;
+
+  return (
+    <section className="comment-list">
+      <h3 className="comment-list-title">Comments</h3>
+      <ul className="comment-card-list">
+        {commentList.map((comment) => {
```

Comment:
There's an error in the console regarding no keys for comments as children of this map. Please add a key